src/ducks/Watchlists/Widgets/Table/CompareInfo/Actions/Actions.module.scss

```diff
@@ -0,0 +1,10 @@
+@import '~@santiment-network/ui/mixins';
+
+.actions {
+    margin-right: 24px;
+    line-height: 0;
+  
+    & svg {
+      cursor: pointer;
```

Comment:
Better to move it to the class like `.action` (which is independent of the `.actions` selector – not a child of it)